cmd/bclient/bserver/chunkfile.go

```diff
@@ -0,0 +1,58 @@
+package bserver
+
+import (
+	"crypto/md5"
+	"fmt"
+	"io"
+	"os"
+	"path"
+)
+
+const BogusFileId string = "Uninitialized"
+
+func chunkAndUpload(fileroot string, srcFile string, srcFileMd5 []byte, item string, fileChunkSize int) (string, error) {
+
+	sourceFile, err := os.Open(path.Join(fileroot, srcFile))
+
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+
+	defer sourceFile.Close()
+
+	chunk := make([]byte, fileChunkSize)
+
+	var fileId string = BogusFileId
```

Comment:
I can't have any fun around you, can I?

On Fri, Jan 22, 2016 at 2:06 PM, Don Brower <notifications@github.com>
wrote:

> In cmd/bclient/bserver/chunkfile.go
> <https://github.com/ndlib/bendo/pull/56#discussion_r50576664>:
>
> > +const BogusFileId string = "Uninitialized"
> > +
> > +func chunkAndUpload(fileroot string, srcFile string, srcFileMd5 []byte, item string, fileChunkSize int) (string, error) {
> > +
> > +	sourceFile, err := os.Open(path.Join(fileroot, srcFile))
> > +
> > +	if err != nil {
> > +		fmt.Println(err)
> > +		os.Exit(1)
> > +	}
> > +
> > +	defer sourceFile.Close()
> > +
> > +	chunk := make([]byte, fileChunkSize)
> > +
> > +	var fileId string = BogusFileId
>
> Why not just use the empty string "" to indicate whether the fileId has
> been assigned yet?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ndlib/bendo/pull/56/files#r50576664>.
>